doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -235,6 +235,8 @@ Bluetooth mesh samples
 
   * Fixed an issue with the sample not fitting into RAM size on the nrf52dk_nrf52832 board.
 
+  * Fixed an issue where samples copied via the VS Code extension did not compile due to relative paths in CMakeLists, referencing files outside the applications folder.
```

Comment:
This should be pulled in with 2 spaces, as it does not have anything to to with sensor_client.